### ISSUE TITLE
Blog images should be marked decorative

### ISSUE THREAD

**Issue by buddy-harlow** [Bixal/bixal-site-drupal#244]
## Location: https://www.bixal.com/blog
When testing with a screen reader, decorative images were announced, example in the screenshot as follows.

## Definition of done
- As a screen reader user, I expect all decorative images (images that are not meaningful) to be hidden from assistive technologies.
- Therefore, all of the Blog graphics should be marked as decorative, UNLESS, the graphic is meaningful and adding content/context.
